src/main/java/org/primeframework/mvc/sampleapp/OurModule.java

```diff
@@ -3,12 +3,13 @@
 import com.google.inject.AbstractModule;
 import org.primeframework.mvc.config.MVCConfiguration;
 import org.primeframework.mvc.cors.CORSConfigurationProvider;
+import org.primeframework.mvc.sampleapp.session.OurLoginContext;
 import org.primeframework.mvc.security.NoLoginSecurityContext;
 import org.primeframework.mvc.security.UserLoginSecurityContext;
 
 public class OurModule extends AbstractModule {
     protected void configure() {
-        bind(UserLoginSecurityContext.class).to(NoLoginSecurityContext.class);
+        bind(UserLoginSecurityContext.class).to(OurLoginContext.class);
```

Comment:
For most of our projects, the top level module is named `FusionAuthModule` or `CleanSpeakModule`. 

So if this isn an example app, should we standardize on `ExampleModule` and `ExampleLoginContext` etc?

Reply:
Done